metadata_service/proxy/neo4j_proxy.py

```diff
@@ -965,3 +967,54 @@ def delete_table_relation_by_user(self, *,
             raise e
         finally:
             tx.close()
+
+    @timer_with_counter
+    def get_dashboard(self,
+                      id: str,
+                      ) -> DashboardDetailEntity:
+
+        get_dashboard_detail_query = textwrap.dedent(u"""
+        MATCH (d:Dashboard {key: $query_key})-[:DASHBOARD_OF]->(dg:Dashboardgroup)-[:DASHBOARD_GROUP_OF]->(c:Cluster)
+        OPTIONAL MATCH (d)-[:DESCRIPTION]->(description:Description)
+        OPTIONAL MATCH (d)-[:EXECUTED]->(last_exec:Execution) WHERE split(last_exec.key, '/')[5] = '_last_execution'
```

Comment:
what is this line for?

Reply:
Assuming the question is about the `WHERE` clause, it's trying to get last execution. There are currently two different executions. One is last execution and the other is last **successful** execution.
https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/models/dashboard/dashboard_execution.py#L23